app/admin/dashboard.rb

```diff
@@ -27,6 +27,17 @@
           button_to "Go into site-wide read-only mode", toggle_read_only_mode_admin_site_settings_path
         end
       end
+      para do
+        form action: update_maximum_concurrent_scrapers_admin_site_settings_path, method: :post do
+          div do
+            label 'Maximum concurrent scrapers'
```

Comment:
Could we add some advice to admins about what the recommended value should be? Whenever I've set this I have to get out a pen and paper to work out what I think it should be.